landingzones/views.py

```diff
@@ -221,6 +222,49 @@ def submit_create(
             raise ex
 
 
+class ZoneUpdateMixin(ZoneConfigPluginMixin):
```

Comment:
To keep with SODAR and SODAR Core tradition, I'd like you to combine this with `ZoneCreateMixin` into `ZoneModifyMixin`. It's logical to have a single helper mixin for both create and update actions. They can then potentially share the same helper methods etc.